build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+    ext {
+        projectGroupName 	= 'com.epam.repair'
+        projectVersion 		= '1.0-SNAPSHOT'
+        mysqlConnectorJavaVersion = '8.0.18'
+        springBootVersion 	= '2.2.0.RELEASE'
+        postgreSqlVersion = '42.2.8'
+        junitVersion 		= '4.12'
+        restAssuredVersion 	= '2.9.0'
+    }
+
+    repositories {
+        mavenCentral()
+        maven { url "https://repo.spring.io/snapshot" }
+        maven { url "https://repo.spring.io/milestone" }
+    }
+
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.2.0.RELEASE")
+    }
+}
+
+allprojects {
+    group projectGroupName
+    version projectVersion
+}
+
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'maven'
+    apply plugin: 'idea'
+    apply plugin: 'jacoco'
+    apply plugin: 'io.spring.dependency-management'
+
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+
+    repositories {
+        mavenCentral()
+        maven { url "https://repo.spring.io/snapshot" }
+        maven { url "https://repo.spring.io/milestone" }
+    }
+
+    dependencyManagement {
+        imports {
+            mavenBom("org.springframework.boot:spring-boot-dependencies:2.2.0.RELEASE")
+        }
+    }
+
+    dependencies {
+        compile "org.springframework.boot:spring-boot-starter"
+        compile "org.springframework.boot:spring-boot-starter-data-jdbc"
+        compile "org.springframework.boot:spring-boot-starter-data-jpa"
+        compile "org.springframework.boot:spring-boot-starter-thymeleaf"
+        compile "org.springframework.boot:spring-boot-starter-web"
+        compile "org.springframework.boot:spring-boot-starter-web-services"
+        compile "com.h2database:h2:"
+        compile "org.postgresql:postgresql:${postgreSqlVersion}"
+        compile "mysql:mysql-connector-java:${mysqlConnectorJavaVersion}"
```

Comment:
1. maybe parent should resolve version?
2. This is a subproject dependency section. Why do you think that all subprojects should have these dependencies?

Reply:
should I put this dependencies to dao-api?

Reply:
`-api`? Why API?